index.html

```diff
@@ -14,7 +14,10 @@
     <script src="bower_components/momentjs/min/moment.min.js"></script>
     <script src="bower_components/momentjs/min/locales.min.js"></script>
     <script src="bower_components/humanize-duration/humanize-duration.js"></script>
-    <script src="dist/angular-timer.js"></script>
+    <!--<script src="dist/angular-timer.js"></script>-->
```

Comment:
Sure... Sounds good to me :)
On Thu, Mar 26, 2015 at 4:37 PM Paul Leclercq <notifications@github.com>
wrote:

> In index.html
> <https://github.com/siddii/angular-timer/pull/171#discussion_r27258707>:
>
> > @@ -14,7 +14,10 @@
> >      <script src="bower_components/momentjs/min/moment.min.js"></script>
> >      <script src="bower_components/momentjs/min/locales.min.js"></script>
> >      <script src="bower_components/humanize-duration/humanize-duration.js"></script>
> > -    <script src="dist/angular-timer.js"></script>
> > +    <!--<script src="dist/angular-timer.js"></script>-->
>
> @siddii <https://github.com/siddii> I think I am going to create a
> angular-timer-all.js for index.html that could be used in the gh-pages and
> for #168 <https://github.com/siddii/angular-timer/issues/168>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/siddii/angular-timer/pull/171/files#r27258707>.
>